akka-http-core/src/main/java/akka/http/javadsl/model/HttpEntity.java

```diff
@@ -40,48 +40,48 @@
     /**
      * Returns the content-type of this entity
      */
-    public abstract ContentType contentType();
+    ContentType contentType();
 
     /**
      * The empty entity.
      */
-    public static final HttpEntityStrict EMPTY = HttpEntity$.MODULE$.Empty();
+    HttpEntityStrict EMPTY = HttpEntity$.MODULE$.Empty();
```

Comment:
I must say I hate java code cause it don't have defined side variance,and I know your pain,cause we have a hyper production codebase too:(